exercises/concept/squeaky-clean/tests/Tests.elm

```diff
@@ -0,0 +1,161 @@
+module Tests exposing (tests)
+
+import Expect
+import SqueakyClean exposing (clean1, clean2, clean3, clean4, clean5)
+import Test exposing (Test, describe, test)
+
+
+tests : Test
+tests =
+    describe "Squeaky Clean"
+        [ describe "Task 1"
```

Comment:
```suggestion
        [ describe "1"
```
As well as for the other tasks.
 
This is required to extract the task ids properly.
We should probably add a condition in the test runner that would fail if a concept exercise is missing ids. [I opened an issue](https://github.com/exercism/elm-test-runner/issues/50).